cmd/crictl/image.go

```diff
@@ -155,6 +155,10 @@ var listImageCommand = &cli.Command{
 			Name:  "no-trunc",
 			Usage: "Show output without truncating the ID",
 		},
+		&cli.BoolFlag{
```

Comment:
why wouldn't we just output it by default without an explicit flag?

Reply:
I didn't think this field was that important, so I left the default output format unchanged.

Of course I'm not particularly sure, I'll take everyone's advice on this